source/basic_skills.rst

```diff
@@ -1304,32 +1304,595 @@ If you managed to solve this, congratulations!
 You are well on your way to becoming a master formalizer.
 
 
-.. .. _using_more_theorems_and_lemmas:
+.. _more_examples:
 
-.. Using More Theorems and Lemmas
-.. ------------------------------
+More Examples
```

Comment:
I think this title is a bit misleading. It suggests that impatient readers could skip this section. But there are several new things introduced here, as well as important preparation for lattices.

Reply:
I'll change it.